manualdirectdebit.php

```diff
@@ -148,12 +148,21 @@ function manualdirectdebit_civicrm_navigationMenu(&$menu) {
   _manualdirectdebit_civix_insert_navigation_menu($menu, 'Administer/' . $directDebitMenuItem['name'], $directDebitCodeSubMenuItem);
 
   $directDebitConfigurationSubMenuItem = [
-    'name' => 'Direct Debit Configuration',
+    'name' => ts('Direct Debit Configuration'),
     'url' => 'civicrm/admin/direct_debit_configuration',
     'permission' => 'administer CiviCRM',
     'operator' => NULL,
     'separator' => NULL,
   ];
   _manualdirectdebit_civix_insert_navigation_menu($menu, 'Administer/' . $directDebitMenuItem['name'], $directDebitConfigurationSubMenuItem);
+
```

Comment:
this is going a bit repeative, you can leave this part as is : 

```php
  $directDebitMenuItem = [
    'name' => ts('Direct Debit'),
    'url' => NULL,
    'permission' => 'administer CiviCRM',
    'operator' => NULL,
    'separator' => NULL,
  ];
  _manualdirectdebit_civix_insert_navigation_menu($menu, 'Administer/', $directDebitMenuItem);
```

but adding the other sub menus can be done in a loop : 

```php
$subMenuItems = [
    'name' => ts('Direct Debit Codes'),
    'url' => 'civicrm/admin/options/direct_debit_codes',
    'permission' => 'administer CiviCRM',
    'operator' => NULL,
    'separator' => NULL,
  ],
[
    'name' => ts('Direct Debit Configuration'),
    'url' => 'civicrm/admin/direct_debit_configuration',
    'permission' => 'administer CiviCRM',
    'operator' => NULL,
    'separator' => NULL,
  ] ..etc

foreach($subMenuItems as $menuItem) {
  _manualdirectdebit_civix_insert_navigation_menu($menu, 'Administer/' . $directDebitMenuItem['name'], $menuItem);
}
```